docs/source/index.rst

```diff
@@ -36,6 +36,36 @@ user:
     print(kennethreitz.followers_count)
 
     followers = [str(f) for f in gh.followers('kennethreitz')]
+    
+    # create a repository
+    gh.create_repository(name='Repo_Name')
+    
+    # grab the repo
+    repos = list(gh.repositories())
+    my_repo = gh.repository(gh.me(), 'Repo_Name') # grab match
+    
+    # create a label
+    my_repo.create_label(name='Ugly', color='d73a4a')
+    
+    # create an issue
+    my_repo.create_issue(title='Problem with Blah Blah')
+    
+    # create a milestone
+    my_repo.create_milestone("7.0.1", due_on='2018-08-21T23:59:59Z')
+    
+    # create a project and add columns to project
+    my_repo.create_project('Project 1')
```

Comment:
Again, we return the created project, I thought, so why are we iterating over them below to find it again?